### ISSUE TITLE
Incorporation of dataset accessibility metadata from WG7

### ISSUE THREAD

**Issue by jgrethe** [biocaddie/WG3-MetadataSpecifications#19]
Since WG7 is finished with their work on the accessibility metadata - this should be included at the dataset level.

**Comment by agbeltran** [comment]
Yes, thank you. We are adding it in the v1.1.

**Comment by jgrethe** [comment]
IS this being added directly into the dataset element?

**Comment by agbeltran** [comment]
That was the plan. Do you have a different suggestion?
